### PR TITLE
Updated create-custom-views-of-native-objects to include the way to install natvis files via VSIX 

### PR DESCRIPTION
**Pull request by xisui-MSFT**
Updated create-custom-views-of-native-objects to include the VSIX manifest NativeVisualizer asset as a method to install a natvis file

### CODE REVIEW

**Comment by PRMerger16** [comment]
@xisui-MSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/debugger/create-custom-views-of-native-objects.md [hunk 1]
```diff
@@ -88,6 +88,30 @@ The Visual Studio debugger loads *.natvis* files in C++ projects automatically,
 >[!NOTE]
 >Natvis rules loaded from a *.pdb* apply only to the types in the modules that the *.pdb* refers to. For example, if *Module1.pdb* has a Natvis entry for a type named `Test`, it only applies to the `Test` class in *Module1.dll*. If another module also defines a class named `Test`, the *Module1.pdb* Natvis entry does not apply to it.
 
+**To install and register a *.natvis* file via a VSIX package:**
+
+A VSIX package can install and register *.natvis* files. No matter where are they installed, all registered *.natvis* files are automatically picked up during debugging.
+
+1. Include the *.natvis* file in the VSIX package. For example in a project file:
```
Comment (Mikejo5000) [inline docs/debugger/create-custom-views-of-native-objects.md]:
```suggestion
1. Include the *.natvis* file in the VSIX package. For example, for the following project file:
```

**Comment by PRMerger9** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
#sign-off Thanks!

File: docs/debugger/create-custom-views-of-native-objects.md (hunk 1 above)
Comment (xisui-MSFT) [reply docs/debugger/create-custom-views-of-native-objects.md]: This change feels weird. I just want to give an example to show that the natvis file needs to be included in a VSIX. There are other ways to do so.

Comment (xisui-MSFT) [reply docs/debugger/create-custom-views-of-native-objects.md]: Actually never mind. Feels fine after read it again.

**Pull request merged**